etc/profile-a-l/firefox-common.profile

```diff
@@ -65,4 +65,5 @@ blacklist ${PATH}/wget2
 # 'dbus-user none' breaks various desktop integration features like global menus, native notifications,
 # Gnome connector, KDE connect and power management on KDE Plasma.
 dbus-user none
+# comment next line to use bluez-alsa.
```

Comment:
I didn't followed the bluzed-alsa discussion, but is this a Firefox specific problem or general for all programs (i.e. should go into the wiki page)?

Reply:
my bad,
even tho I've only tried it with firefox
it shouldn't be firefox specific

Reply:
IMHO  https://github.com/netblue30/firejail/wiki/Troubleshooting#common-problems is then the better place.